packages/x-charts/src/context/ChartProvider/ChartProvider.tsx

```diff
@@ -0,0 +1,15 @@
+'use client';
+import * as React from 'react';
+import { useCharts } from '../../internals/store/useCharts';
+import { ChartProviderProps } from '../ChartProvider.types';
+import { ChartContext } from './ChartContext';
+
+function ChartProvider(props: ChartProviderProps) {
+  const { children } = props;
+
+  const { contextValue } = useCharts([], {});
```

Comment:
Here we will have to propagate the plugins and to props to generate the internal state and add the different methods

Reply:
what do you mean?

Reply:
For example the cartesian chart shoudl enable axis processing with something like

```
<ChartProvider 
  plugins={[
    useCartesianAxes
  ]}
>
```

This plugin would add the x/y axes to the store when doing

```
const { contextValue } = useCharts([useCartesianAxes], { xAxis, yAxis; ... });
```